arbitrum-docs/stylus/reference/testnet-information.md

```diff
@@ -7,23 +7,30 @@ target_audience: Developers building on the Stylus testnet
 sidebar_position: 9
 ---
 
+import PublicPreviewBannerPartial from '../partials/_stylus-public-preview-banner-partial.md';
```

Comment:
Can we remove this now that a Nitro release supporting Stylus in production is going to be activated on ArbOne in <2 weeks?